### ISSUE TITLE
[BUG]: Debugger filter options that don't use the "Value" input still verify the input field

### ISSUE THREAD

**Issue by F0bes** [PCSX2/pcsx2#12007]
### Describe the Bug

When searching by `Increased`,`Decreased`,`Changed` or `Unchanged` the `Value` input box is still validated despite it not being used during the comparison.

### Reproduction Steps

- Do a memory search.
- Set the comparison to one of the ones listed above.
- Clear the value input.
- Try and filter search.
---
You should see a message-box stating "Invalid search value" 

### Expected Behavior

The filter-search should happen normally without issue.

### PCSX2 Revision

v2.3.10

### Operating System

Linux (64bit) - Specify distro below

### If Linux - Specify Distro

Endeavour

### Logs & Dumps

N/A

**Comment by recursean** [comment]
Hey @F0bes do you still get this behavior? I'm unable to reproduce with the most recent build. I think you [fixed](https://github.com/recursean/pcsx2/commit/3e1f2b8b9d3c5c3d9c0b049b6af24410cc3f728c) it back in Feb :) Maybe this issue can be closed?

**Comment by F0bes** [comment]
Yeah I remember fixing it. I didn't realize I had an issue for it. Thanks!
